py/torch_tensorrt/dynamo/lowering/_partition.py

```diff
@@ -164,11 +87,139 @@ def print_support_overview(self, num_trt_blocks: Optional[int] = None):
             logger.debug("\nAll Nodes Supported\n")
 
 
+class TRTPartitioner(_SplitterBase):
```

Comment:
What is the correctness of this Partitioner vs the Capability one? We could keep both behind a flag or something if there is a difference

Reply:
If we want to do this, Id suggest a structure similar to runtime in a submodule

Reply:
Since inductor is on Capability partitioner id expect better maintenance, correctness and improvements to performance

Reply:
Alternatives in pytorch: https://github.com/pytorch/pytorch/blob/df8abaaf5f7e19d897aa30896bdbbe416ca2f825/torch/fx/experimental/accelerator_partitioner.py#L287
https://github.com/pytorch/pytorch/blob/df8abaaf5f7e19d897aa30896bdbbe416ca2f825/torch/fx/passes/infra/partitioner.py#L34

Reply:
Fast vs full partitioning --> for compile, fast fits best, set as default, give users the option to upgrade to full partitioner.